crates/ide_assists/src/handlers/auto_import.rs

```diff
@@ -918,6 +918,9 @@ fn main() {
             auto_import,
             r"
 //- /lib.rs crate:dep
+pub mod formatters;
+
+//- /formatters.rs crate:dep
```

Comment:
Hm, why do we need an explicit crate here? I would expect that everything after first crate goes to the same crate by default.

Reply:
That's exactly how it works nowadays, I've removed the confusing crate declaration 🙂 

One good point is to actually allow to omit the dependencies, I've adjusted that.